assets/app/views/settings.html

```diff
@@ -0,0 +1,83 @@
+<div ng-controller="ProjectController">
+  <project-page>
+    <div ng-controller="SettingsController">
+      <h1>Project Settings</h1>
+      <alerts alerts="alerts"></alerts>
+      <h2>General information</h2>
+      <!-- TODO make these fields editable - probably with a cog icon that flips into inline-editing -->
+      <dl class="dl-horizontal left">
+        <dt>Display name</dt>
+        <dd ng-if="project.displayName">{{project.displayName}}</dd>
+        <dd ng-if="!project.displayName"><em>No display name</em></dd>
+        <dt>Description</dt>
+        <dd ng-if="project.metadata.annotations && project.metadata.annotations.description">{{project.metadata.annotations.description}}</dd>
+        <dd ng-if="!project.metadata.annotations || !project.metadata.annotations.description"><em>No description</em></dd>
+      </dl>
+      <h2>
+        <span>Quota usage</span>
+        <a href="javascript:;" style="margin-left: 5px; cursor: pointer;" class="small pficon pficon-help" data-container="body" data-toggle="popover" data-placement="right" data-trigger="focus" data-content="Quota usage is aggregated for the whole project.  Once the maximum for the project has been reached no more of that resource can be allocated within the project."></a>
```

Comment:
Does this render if and only if a quota is on for the project?

The message looks fine.